app/views/pages/internet_access.html.erb

```diff
@@ -15,6 +15,10 @@
       How to request internet access for disadvantaged pupils in <span class="app-no-wrap">years 3 to 13</span>, students in further education, and care leavers.
     </p>
 
+<div class="govuk-inset-text">
+      Young people and families on low incomes or certain benefits may be eligible for cheaper broadband tariffs. <%= govuk_link_to 'Find out who is eligible and which providers offer targeted tariffs', 'https://www.ofcom.org.uk/about-ofcom/latest/features-and-news/support-for-customers-during-pandemic' %>
```

Comment:
Should "Find out who is eligible and which providers offer targeted tariffs" have a full stop (outside the link?) sorry I know this is my fault.

Reply:
Yes, thanks for spotting!

Reply:
You're right @ralph-hawkins - don't apologise!